NeoML/src/Dnn/Dnn.cpp

```diff
@@ -497,6 +497,30 @@ void CDnn::ForceRebuild()
 	sourceLayers.SetSize( 0 );
 }
 
+CDnn* CDnn::CreateReferenceDnn(CRandom& random)
+{
+	reshape();
```

Comment:
Add `NeoAssert` to restrict to learn dnn, if its reference dnns existed
Add `NeoAssert` to restrict to update reference blob